pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -244,6 +274,50 @@ func (br *OVSBridge) SetExternalIDs(externalIDs map[string]interface{}) Error {
 	return nil
 }
 
+// SetDatapathID sets the provided datapath ID to the bridge.
+// Whenever a bridge reconfigures (ports are added or /removed) and pick a different MAC address
```

Comment:
This is Windows specific?
What is datapath ID? Why we need to set/get it?

I did not get this sentence:
> Whenever a bridge reconfigures (ports are added or /removed) and pick a different MAC address and has not a datapath ID configured.
Datapath changes when all the 3 conditions are met: reconfigures, picks a different MAC, and has no datapath ID configured before (if no datapath ID before why it is a change?); or when any of the 3 conditions is met?
What does "picks a different MAC" mean? Maybe you should rephrase.

%s/pick/picks

Reply:
Update the comment.